src/past/translation/__init__.py

```diff
@@ -219,20 +220,20 @@ def detect_python2(source, pathname):
     if source != str(tree)[:-1]:   # remove added newline
         # The above fixers made changes, so we conclude it's Python 2 code
         logger.debug('Detected Python 2 code: {0}'.format(pathname))
-        with open('/tmp/original_code.py', 'w') as f:
+        with open(os.path.join(temppath,'original_code.py'), 'w') as f:
```

Comment:
I recommend using `tempfile.mkdtemp` instead of this and removing line 211 (`temppath`)

Also fixes bug related to simultaneous use of `past` on the same system